### PR TITLE
Copie of form web app 

### PR DESCRIPTION
**Pull request by miguelesco**
Copie of microverse test

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    font-size: 1.125rem;
+    margin-bottom: 0.5rem;
+}
+
+.input-textarea {
+    min-height: 120px;
+    width: 100%;
+    padding: 0.625rem;
+    resize: vertical;
+}
```
Comment (miguelesco) [inline css/style.css:71]: please remember to place the linter in the project.  So you can avoid programming errors, bugs, stylistic errors, and suspicious constructs

File: index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (miguelesco) [inline index.html:94]: please remove comment lines. So you can keep your code cleaner

File: index.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    <div style="width: 50vw; margin: 0 auto;">
```
Comment (miguelesco) [inline index.html:13]: considers using the main tag as a general container. so you restructure can be more easily understood and clean

File: index.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```
Comment (miguelesco) [inline index.html:10]: considers place the script tag at the end of the body for a better clean code

File: index.html [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+    
+    <div style="width: 50vw; margin: 0 auto;">
+      
+      <h1 id="title" style="text-align: center; margin: auto;">Title</h1>
```
Comment (miguelesco) [inline index.html:15]: please place the style of the elements in the CSS file and use class to identify the styles of the elements in the HTML

File: index.html [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+        <div class="form-group">
+          <textarea name="comments" class="input-textarea" placeholder="enter your comments here..."></textarea>
+        </div>
+      
+
+        <div class="form-group">
+          <input type="submit" id="submit" class="submit-button">
+        </div>
+      </form>
```
Comment (miguelesco) [inline index.html:76]: considers encapsulates the form tag in a section tag

File: README.md [hunk 7]
```diff
@@ -45,15 +45,9 @@ To get a local copy up and running follow these simple example steps.
...
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@miguelesco](https://github.com/miguelesco)
+- Twitter: [@miguelescorched](https://twitter.com/miguelescorched)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/miguelangel-escorche-delgado-9a2956163/)
```
Comment (miguelesco) [inline README.md:51]: remember to fill all the info of the project in the readme for an easy understanding of your project. And also remember to mention your partner that work with you in this project  in order to comply with the policys of plagiarism

**Review by miguelesco — COMMENTED** [review]
# Changes required 🧷

You are doing great 😄 but I'm sure you can improve your code only by following these little changes.

Please check my comments Below and make all the required changes.

Submit for a re-review after you finish.

**Review by SpaYco — APPROVED** [review]
Hi @miguelesco, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**